src/routes/About.tsx

```diff
@@ -19,6 +19,9 @@ export default function About(): JSX.Element {
         </a>{' '}
         mit dem Stand vom 12.01.2022.
       </p>
+      <p>
+        Ein Test wird mit einem grünen Haken angezeigt, wenn er Infizierte mit sehr hoher Viruslast zu über 75% erkennt.
+      </p>
```

Comment:
The green checkmark is based on the minimum criteria the PEI uses, which should be reflected in this text. How about something like this:

```suggestion
            <p>
        Ein Test wird mit einem grünen Haken angezeigt, wenn die Sensitivität bei sehr hoher
        Viruslast (Cq ≤ 25) mehr als 75% beträgt. Dies entspricht dem vom{' '}
        <a href="https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/evaluierung-sensitivitaet-sars-cov-2-antigentests.pdf?__blob=publicationFile">
          Paul-Ehrlich-Institut angenommenen "Stand der Technik".
        </a>
      </p>
```